cypress/support/commands.js

```diff
@@ -23,3 +23,18 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (user) => {
+  const url = 'https://demoqa.com/Account/v1/Login';
+  cy.request('POST', url, {
+    userName: user.username,
+    password: user.password
+  }).then((response) => {
+    cy.setCookie('token', response.body.token);
+    cy.setCookie('userID', response.body.userId);
+    cy.setCookie('userName', response.body.username);
+    cy.setCookie('expires', response.body.expires);
+
+    cy.visit('/profile');
```

Comment:
I think its extra command to "login", because if u want to use it later - you always be redirects to profile page